examples/advanced/src/example.tsx

```diff
@@ -828,7 +830,7 @@ export default class Example extends React.Component<
 
   renderStaticMap(viewport: Record<string, any>) {
     return (
-      // @ts-ignore
+      // @ ts-ignore
```

Comment:
Looks like `@ts-ignore ` it was right
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html

Reply:
ignore not needed, removed